reporter/build.gradle.kts

```diff
@@ -60,6 +62,36 @@ repositories {
             includeGroup("bad.robot")
         }
     }
+
+    exclusiveContent {
+        forRepository {
+            maven("https://download.eclipse.org/antenna/releases/")
+        }
+
+        filter {
+            includeGroup("org.eclipse.sw360.antenna")
+        }
+    }
+
+    exclusiveContent {
+        forRepository {
+            maven("https://jitpack.io")
+        }
+
+        filter {
+            includeGroup("com.github.ralfstuckert.pdfbox-layout")
```

Comment:
https://github.com/ralfstuckert/pdfbox-layout has not received any updates for the last 3 years seems like a dead OSS project. There are several forks ...

Reply:
Unfortunately there is no alternative, but we will try to get rid of the dependency later on.

Reply:
FYI, even *pdfbox* itself still [recommends pdfbox-layout](https://pdfbox.apache.org/2.0/faq.html#can-i-use-pdfbox-to-create-complex-layouts). I've checked the other mentioned projects, but none of them really is an alternative to *pdfbox-layout* feature-wise.

I agree it's not so nice that we have to add JitPack here, but in this case the fact that there haven't been updates in 3 year does not seem to have a negative impact on the project's quality.